cgroup_top.py

```diff
@@ -100,6 +103,60 @@ def cmd_exists(cmd):
 
 ## Utils
 
+
+def strip_prefix(prefix, text):
+    if text.startswith(prefix):
+        return text[len(prefix):]
+    return text
+
+
+def docker_container_name(container_id, default, cache=dict()):
+    # Python’s default arguments are evaluated when the function is
+    # defined, not when the function is called.
+    # We potentially cache and return a default value so we don't spend time
+    # pointlessly retrying to get the container name if something goes wrong.
+    cached_name = cache.get(container_id)
+    if cached_name:
+        return cached_name
+
+    try:
+        sp = subprocess.Popen(['docker', 'inspect', container_id],
+                              stdout=subprocess.PIPE,
+                              stderr=subprocess.PIPE)
+    except OSError:
```

Comment:
Actually, with the ``HAS_DOCKER`` check lower, this should never trigger. But that's harmless / future proof.